src/main/java/org/sonar/plugins/stash/issue/MarkdownPrinter.java

```diff
@@ -31,30 +43,38 @@ public static String printSeverityMarkdown(String severity) {
     return sb.toString();
   }
 
-  public static String printIssueNumberBySeverityMarkdown(SonarQubeIssuesReport report, CoverageIssuesReport coverageReport, String severity) {
+  public static String printIssueNumberBySeverityMarkdown(List<Issue> report, String severity) {

```

Comment:
`List<Issue> report` => `List<Issues> issues` (wrong param name now)

But actually, passing the report is OK as for me